.github/workflows/ci.yml

```diff
@@ -19,4 +19,4 @@ jobs:
           java-version: '17'
           cache: 'gradle'
       - name: Build
-        run: ./gradlew check
+        run: ./gradlew check -x test -x integration -x pmdTest -x pmdFunctionalTest -x pmdIntegrationTest
```

Comment:
What's the idea behind skipping pmd? If we are skipping pmd should we skip checkstyle too?

Reply:
To be fair I just copied what was done as a part of the spike in DMP-1661 but I believe this was added because we didn't need to replicate the same checks that are done on jenkins within github environment. I guess keeping checkstyle in means the build can fail quicker on this kind of issue @johnrace0?

Reply:
the idea was to just skip all the bits that fail on github